File: component-service/src/main/java/dev/yidafu/pan/component/NonstickPanComponentApplication.java
package dev.yidafu.pan.component;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("dev.yidafu.pan.component")
public class NonstickPanComponentApplication {

	public static void main(String[] args) {
		SpringApplication.run(NonstickPanComponentApplication.class, args);
	}
}

File: component-service/src/main/kotlin/dev/yidafu/pan/component/NonstickPanComponentApplication.kt
package dev.yidafu.pan.component

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


/**
 * https://blog.csdn.net/weixin_43118901/article/details/109153215
 */
@SpringBootApplication
class DemoApplication

fun main(args: Array<String>) {
    runApplication<DemoApplication>(*args)
}
